scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/validation/RepositoryUrlConstraintValidator.java

```diff
@@ -0,0 +1,28 @@
+package edu.asu.diging.scriptoocloud.web.validation;
+
+import java.net.URL;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RepositoryUrlConstraintValidator implements ConstraintValidator<RepositoryUrlConstraint, String> {
+
+    private Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Override
+    public void initialize(RepositoryUrlConstraint constraintAnnotation) {}
+
+    public boolean isValid(String urlField, ConstraintValidatorContext context) {
+        boolean valid = true;
+        try {
+            URL url = new URL(urlField);
+        } catch(final Exception ignore){
```

Comment:
why is that final?

Reply:
why do you catch `Exception` here? It's throwing a `MalformedURLException`, doesn't it?